packages/webapp/components/layouts/AccountLayout/index.tsx

```diff
@@ -28,12 +29,23 @@ export const navigationKey = generateQueryKey(
 export default function AccountLayout({
   children,
 }: AccountLayoutProps): ReactElement {
-  const { user: profile, isFetched, logout } = useContext(AuthContext);
+  const router = useRouter();
+  const { user: profile, isFetched } = useContext(AuthContext);
+  const isMobile = useViewSize(ViewSize.MobileL);
   const [isOpen, setIsOpen] = useQueryState({
     key: navigationKey,
     defaultValue: false,
   });
-  const isMobile = useViewSize(ViewSize.MobileL);
+
+  useEffect(() => {
```

Comment:
This feels a bit wonky (prone to issues, but don't have a good alternative)
CC: @capJavert